public/check

```diff
@@ -0,0 +1 @@
+OK
```

Comment:
can we drop the SHA here instead (the command you dropped re docker)

Reply:
will do that as a separate PR, 
we also have `/integrations/monitoring/version` for the version check for the time being